pipeline/alert.go

```diff
@@ -355,6 +361,7 @@ func newAlertNode(wants EdgeType) *AlertNode {
 		Id:        defaultIDTmpl,
 		Message:   defaultMessageTmpl,
 		Details:   defaultDetailsTmpl,
+		Hostname:  defaultHostnameTmpl,
```

Comment:
Again instead of making this a global option to all alerts add it the the SensuHandler type which can be found later on in this file.